cl/users/api_views.py

```diff
@@ -62,6 +63,8 @@ def create(self, request, *args, **kwargs):
         if form.is_valid():
             webhook.user = request.user
             form.save()
+            # Send webhook created notification to admins
+            notify_new_or_updated_webhook.delay(webhook.pk)
```

Comment:
Would a signal be better? I'm not sure. Curious your opinion.

Reply:
Yeah, I think a signal works well here since we are going to centralize the trigger instead of adding to the views.
So changed and also added a parameter to check if the webhook is being created or updated to show in the notification.